src/HasApiTokens.php

```diff
@@ -45,9 +45,16 @@ public function tokenCan(string $ability)
      */
     public function createToken(string $name, array $abilities = ['*'], DateTimeInterface $expiresAt = null)
     {
+        $plainTextToken = sprintf(
+            '%s%s%s',
+            config('sanctum.token_prefix', ''),
```

Comment:
This will never be empty, because the vendor's sanctum.php defines a `laravel_sanctum_` default value, regardless if the env var is defined or not.

If I understand correctly, this has been included in a minor version release 3.2.6. We use Laravel Sanctum version 3.2. We accidentally realized the tokens now have a "laravel_sanctum_" prefix unless the env var is redefined with empty content, which is unexpected.

How is this not a breaking change?

Thanks for any feedback you may have.

Reply:
Thanks @driesvints!

Reply:
Thank you! I'm afk, but based on the change in the code the unexpected change in behavior should be gone.